org.gnome.PasswordSafe.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "org.gnome.PasswordSafe",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-passwordsafe",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=/usr/share/icons:ro",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=home",
```

Comment:
Why does a password manager need home access?

Reply:
To open the keepass database files I guess

Reply:
GtkFileChooserNative should be good enough for single files.

Reply:
GtkFileChooserNative has some issues with python bindings as far as I know. The last time I have tried it, it wasn't working correctly :/

Reply:
For opening database files and for real paths (/home/user/...) instead of (/run/...)

Reply:
@bilelmoussaoui That makes absolutely no sense, but if true report a bug and link me =)

Reply:
> For opening database files and for real paths (/home/user/...) instead of (/run/...)
 
Why do you care what the path is?

Reply:
Because it is displayed in the application

Reply:
![path](https://user-images.githubusercontent.com/5732964/46361786-c6465380-c66e-11e8-99e5-dea542f91daa.png)

Reply:
@TingPing Will try to reproduce in a clean env and open an issue if I still have the same issue

Reply:
@bilelmoussaoui https://github.com/flatpak/flatpak/issues/2177

Reply:
Hmm, while I agree it would be nice to have a full path I think the security gain from not having filesystem access is much more important than that UI detail. For now you could just only show `Passwords.kdbx` in the header.

Reply:
Removed the permission, I'll see what I can do about the path later.
Seems like there were efforts to achieve showing real paths from inside the sandbox https://github.com/flatpak/flatpak/pull/1335